TCSA.V2/Helpers/ProjectsSubHelpers/ReactProjectsHelper.cs

```diff
@@ -4,6 +4,15 @@
 namespace TCSA.V2.Helpers.ProjectsSubHelpers;
 internal static class ReactProjectsHelper
 {
+    private static readonly IConfiguration Configuration;
+    private static readonly string DiscordLink;
+
+    static ReactProjectsHelper()
+    {
+        Configuration = ServiceProviderAccessor.ServiceProvider.GetService<IConfiguration>();
+        DiscordLink = Configuration["LinkProvider:DiscordLink"];
```

Comment:
```suggestion
        configuration = ServiceProviderAccessor.ServiceProvider.GetService<IConfiguration>();
        discordLink = Configuration["LinkProvider:DiscordLink"];
```

brings in line with other project files